### PR TITLE
Add npm lint script and fix some files.

### PR DESCRIPTION
**Pull request by hblumberg**
Just a little clean up! There should be no functional changes.

In follow up PRs, I'll ensure all JavaScript files are linted.

### Test Plan:
* Run `npm run lint` and see no files changes or errors reported.
* Ensure the website functions as usual.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,8 @@
   },
   "scripts": {
     "build": "next build",
-    "local": "eslint app.js db/*.js geocode/*.js routes/*.js utils/*.js  --fix & node app.js",
+    "lint": "eslint *.js db/*.js routes/*.js utils/*.js --fix",
+    "local": "eslint *.js db/*.js routes/*.js utils/*.js --fix && node app.js",
```
Comment (johndeyrup) [inline package.json]: looks good I updated the lint file to be this in another pr

**Review by mwellman17 — APPROVED** [review]

**Review by johndeyrup — APPROVED** [review]
